sdk/core/System.ServiceModel.Rest/src/System.ServiceModel.Rest.csproj

```diff
@@ -2,15 +2,11 @@
 	<PropertyGroup>
 		<Description>Contains base building blocks for TypeSpec generated libraries.</Description>
 		<Version>1.0.0-beta.1</Version>
-		<!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
 		<Nullable>enable</Nullable>
-		<TargetFrameworks>netstandard2.0;netcoreapp2.1</TargetFrameworks>
-		<IsClientLibrary>true</IsClientLibrary>
+
+		<!-- TODO: Remove netcoreapp2.1 once we understand how to handle NU1605 warning as error -->
+		<TargetFrameworks>netstandard2.0;netcoreapp2.1;net6.0</TargetFrameworks>
```

Comment:
I had to keep netcoreapp2.1 for now, until I can figure out how to resolve this warning:

```
9>C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2364,5): warning MSB3277: Found conflicts between different versions of "System.Diagnostics.DiagnosticSource" that could not be resolved.
9>C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2364,5): warning MSB3277: There was a conflict between "System.Diagnostics.DiagnosticSource, Version=4.0.4.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" and "System.Diagnostics.DiagnosticSource, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51".
9>C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2364,5): warning MSB3277:     "System.Diagnostics.DiagnosticSource, Version=4.0.4.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" was chosen because it was primary and "System.Diagnostics.DiagnosticSource, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" was not.
```

Reply:
I ended up removing the target from Azure.Core and adding a tracking item to validate that's feasible to https://github.com/Azure/azure-sdk-for-net/issues/39023